src/material.js

```diff
@@ -1,7 +1,9 @@
-import shaderSources from './gl/shader_sources'; // built-in shaders
 import GLSL from './gl/glsl';
 import {StyleParser} from './styles/style_parser';
 
+let fs = require('fs');
```

Comment:
Oh yeah this rings a bell. It looks like Matt actually went ahead and made this to work with babel: https://github.com/Jam3/brfs-babel

But since it sounds like this may crop up again, maybe not worth the effort to switch it out just to avoid a little ugliness :)